conans/client/cmd/download.py

```diff
@@ -42,13 +45,21 @@ def download(app, ref, package_ids, remote, recipe, recorder, remotes):
 
 def _download_binaries(conanfile, ref, package_ids, cache, remote_manager, remote, output,
                        recorder, parallel):
-    short_paths = conanfile.short_paths
 
     def _download(package_id):
         pref = PackageReference(ref, package_id)
+        # TODO: cache2.0 here when specifying a package_id with conan download
```

Comment:
But this doesn't look right, when we query to the server for package_ids, don't we get the revisions too?